SocialCareCaseViewerApi.Tests/V1/UseCase/CaseTypesUseCaseTests.cs

```diff
@@ -0,0 +1,37 @@
+using System;
+using System.Collections.Generic;
+using Moq;
+using NUnit.Framework;
+using SocialCareCaseViewerApi.Tests.V1.Helpers;
+using SocialCareCaseViewerApi.V1.Infrastructure;
+using SocialCareCaseViewerApi.V1.Gateways;
+using SocialCareCaseViewerApi.V1.UseCase.Interfaces;
+using FluentAssertions;
+using SocialCareCaseViewerApi.V1.Exceptions;
+
+namespace SocialCareCaseViewerApi.Tests.V1.UseCase
+{
+    public class CaseTypesUseCaseTests
+    {
+        private Mock<IDatabaseGateway> _mockDatabaseGateway;
+        private CaseStatusesUseCase _caseStatusesUseCase;
+
+        [SetUp]
+        public void SetUp()
+        {
+            _mockDatabaseGateway = new Mock<IDatabaseGateway>();
+            _caseStatusesUseCase = new CaseStatusesUseCase(_mockDatabaseGateway.Object);
+        }
+
+        [Test]
+        public void WhenCaseStatusIsNotFoundAndDatabaseGatewayReturnsNullThrowsGetCaseStatusExceptionWithMessage()
```

Comment:
Might want to update test title, throws exception when person is not found

Reply:
Title updated :)